app/models/session.rb

```diff
@@ -26,4 +26,8 @@ class Session < ApplicationRecord
     self.user_agent = Current.user_agent
     self.ip_address = Current.ip_address
   end
+
+  def sign_out_siblings
```

Comment:
Nit: Would it make sense to add a bang here!
```suggestion
  def sign_out_siblings!
```